extensions/amp-carousel/0.1/amp-carousel.css

```diff
@@ -17,6 +17,22 @@
 amp-carousel {
 }
 
+/**
+ * `amp-carousel-slide` class is added to every item in the carousel to
+ * allow page authors to target items for user-defined styling.
+ * Every `<amp-carousel>'s` immediate children is considered an item in
+ * the carousel.
+ */
+amp-carousel-slide {
+}
+
+.amp-carousel-slide .-amp-replaced-content {
```

Comment:
what do you think about doing only 2 (like of an arbitrary number though) levels to cover the common cases yet not going too deep?
like 
`.amp-carousel-slide > .-amp-replaced-content` and
`.amp-carousel-slide > * > .-amp-replaced-content`